crypto/include/internal/ec_int.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+/* Internal EC functions for other submodules: not for application use */
+
+#ifndef HEADER_OSSL_EC_INTERNAL_H
+# define HEADER_OSSL_EC_INTERNAL_H
+# include <openssl/opensslconf.h>
+
+# ifndef OPENSSL_NO_EC
+
+#  include <openssl/ec.h>
+
+/*-
+ * Computes the multiplicative inverse of x in the range
+ * [1,EC_GROUP::order).
+ *
+ *         res := x^(-1) (mod EC_GROUP::order)
+ *
+ * - When the EC group order is even, this function returns an error.
+ * - When the EC group order is otherwise composite, the correctness
+ *   of the output is not guaranteed.
+ * - When x is outside the range [1, EC_GROUP::order), the correctness
+ *   of the output is not guaranteed.
+ */
+int ec_group_do_inverse_ord(const EC_GROUP *group, BIGNUM *res,
```

Comment:
Would it be worth adding `__owur` here? (warn on unused result)

Reply:
I'd say that annotation is a very good idea, always!

I'll look up how you are adding it on other functions to try and be consistent with the code style!

Reply:
Added in the last revision.